packages/integration-tests/src/test-e2e/deployment/cdk.deployment.test.template.ts

```diff
@@ -0,0 +1,50 @@
+import { after, afterEach, before, beforeEach, describe, it } from 'node:test';
+import {
+  createTestDirectory,
+  deleteTestDirectory,
+  rootTestDir,
+} from '../../setup_test_directory.js';
+import { testConcurrencyLevel } from '../test_concurrency.js';
+import { TestCdkProjectBase } from '../../test-project-setup/cdk/test_cdk_project_base.js';
+import { TestCdkProjectCreator } from '../../test-project-setup/cdk/test_cdk_project_creator.js';
+
+/**
+ * Defines cdk deployment test
+ */
+export const defineCdkDeploymentTest = (
```

Comment:
this has been split as is from here https://github.com/aws-amplify/amplify-backend/blob/5f46d8dc1f76d567065b2aa724ec571592bbfa1f/packages/integration-tests/src/test-e2e/deployment.test.ts#L169-L190